### DISCUSSION TITLE
Drastic increase in file size with libvips versions after v8.12.2

### DISCUSSION THREAD

**Discussion by KJ1i** [libvips/libvips#3762] [General]
Hello,

I am writing to report an issue with libvips versions beyond v8.12.2.

**Problem:**

After processing an image, such as this zipped one [input.zip](https://github.com/libvips/libvips/files/13425298/input.zip), using libvips versions newer than v8.12.2, there is a bug causing a drastic increase in the file size of the output image.

**Example:**

- In v8.15.0, the size of the image is escalated from 18.2 MB to 31.7 MB.
- Conversely, in v8.12.2, the increase is comparatively modest, going from 18.2 MB to 19.1 MB.

**Test Command Used:**

`vips copy input.png output.png`

**Steps to Reproduce:**

1. Download the above zipped image file.
2. Use the vips copy command as specified above with versions v8.12.2 and v8.15.0.
3. Observe the resulting file sizes of the output images.

**Expected Result:**

The file size should remain relatively stable across libvips versions, as observed between v8.12.2 and v8.15.0.

Thank you for your time and assistance.

**Comment by lovell** [comment]
Please see https://github.com/libvips/libvips/pull/2929 - you'll need to specify `filter=all` for the same behaviour as v8.12.2.

**Reply by KJ1i** [reply]
Thanks for the quick reply.
